tests/pdf_viewer/test_pdf_download.py

```diff
@@ -75,4 +75,3 @@ def test_pdf_download(
     print(
         f"Test passed: The file {file_name} has been downloaded and is present at {saved_pdf_location}."
     )
-    driver.quit()
```

Comment:
Why remove this?

Reply:
I think linting deleted this which is super weird, reverting now